main.js

```diff
@@ -0,0 +1,37 @@
+console.log('hi');
+
+var number = 1500000
```

Comment:
Right now you're counting down from the right number, but how would you format it so that it displays in minutes and seconds?